### PR TITLE
docs: changing gen1 pubsub links -> gen2

### PR DESCRIPTION
**Pull request by hahnandrew**
#### Description of changes

changing pubsub links from gen1 -> gen2

#### Description of how you validated changes
n/a

#### Checklist

- [x] PR description included

#### Checklist for repo maintainers

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/api/modules/_aws_amplify_pubsub.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...://www.npmjs.com/package/aws-amplify">NPM</a><a href="https://github.com/aws-amplify/amplify-js">GitHub</a></div></div><ul class="results"><li class="state loading">Preparing search index...</li><li class="state failure">The search index is not available</li></ul><a href="../index.html" class="title">Amplify JS API Documentation</a></div><div class="table-cell" id="tsd-widgets"><a href="#" class="tsd-widget tsd-toolbar-icon menu no-caption" data-toggle="menu" aria-label="Menu"><svg width="16" height="16" viewBox="0 0 16 16" fill="none"><use href="#icon-menu"></use></svg></a></div></div></header><div class="container container-main"><div class="col-content"><div class="tsd-page-title"><ul class="tsd-breadcrumb"><li><a href="../index.html">Amplify JS API Documentation</a></li><li><a href="_aws_amplify_pubsub.html">@aws-amplify/pubsub</a></li></ul><h1>Module @aws-amplify/pubsub - v6.1.11</h1></div><section class="tsd-panel tsd-typography"><p>This package contains the AWS Amplify PubSub category. For more information on using PubSub in your application please reference the <a href="https://docs.amplify.aws/javascript/build-a-backend/add-aws-services/pubsub/">Amplify Dev Center</a>.</p>
```
Comment (ashika112) [inline docs/api/modules/_aws_amplify_pubsub.html:2]: was this auto generated?

Comment (hahnandrew) [reply docs/api/modules/_aws_amplify_pubsub.html:2]: no

Comment (hahnandrew) [reply docs/api/modules/_aws_amplify_pubsub.html:2]: no? unless if the question was if i used gnu sed

Comment (hahnandrew) [reply docs/api/modules/_aws_amplify_pubsub.html:2]: if the question wasn't if i used gnu sed, no

**Pull request closed without merging**